core/sql/cli/Context.h

```diff
@@ -505,6 +519,8 @@ class ContextCli : public ExGod {
   // It points to a list of 'black box' of data allocated by user and is returned
   // back whenever this context is in use.
   HashQueue * trafSElist_;
+  // memory quota allocation given back by BMOs to be used by other BMOs
+  ULng32 unusedBMOsMemoryQuota_;
```

Comment:
What unit of measurement is this value? Bytes? KB? MB? If Bytes, is ULng32 big enough?

Reply:
It is in MB.